components/openindiana/illumos-gcc/illumos-gcc.p5m

```diff
@@ -766,20 +766,20 @@ file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSIO
 file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/crtbegin.o
 file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/crtend.o
 file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/gmon.o
-file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/ammintrin.h
-file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/avxintrin.h
-file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/bmmintrin.h
-file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/cpuid.h
-file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/cross-stdarg.h
-file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/emmintrin.h
+file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/ammintrin.h variant.arch=i386
+file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/avxintrin.h variant.arch=i386
+file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/bmmintrin.h variant.arch=i386
+file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/cpuid.h variant.arch=i386
+file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/cross-stdarg.h variant.arch=i386
+file path=opt/gcc/$(COMPONENT_VERSION)/lib/gcc/$(GNU_TRIPLET)/$(COMPONENT_VERSION)/include/emmintrin.h variant.arch=i386
```

Comment:
Should some headers be added with variant sparc?

Reply:
It seems we do have them all.